copyright

```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
It looks like your editor inserted an invisible byte order mark here, which is completely unnecessary for UTF-8 files and will confuse the heck out of certain other text editors.